cmake/cflags.cmake

```diff
@@ -1,7 +1,7 @@
 # Set compiler-specific flags
 # Each of our project dirs sets CMAKE_CXX_FLAGS based on these. We do
 # not set CMAKE_CXX_FLAGS globally because gtest is not warning-clean.
-if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
+if ("${CMAKE_CXX_COMPILER_ID}" MATCHES "\\w*Clang")
```

Comment:
I'll update the message. On Apple the ID changed to AppleClang in a cmake release (not sure which version).